packages/cli/package.json

```diff
@@ -20,7 +20,9 @@
         "build": "tsc && npm run copyfiles",
         "copyfiles": "copyfiles -u 1 lib/templates/* dist",
         "dev": "npm run copyfiles && tsc -w",
-        "prepublishOnly": "npm install && npm run build && cp '../shared/dist/lib/sdk/sync.d.ts' './dist/nango-sync.d.ts' "
+        "postbuild": "cp `node -p \"require.resolve('@nangohq/shared/dist/lib/sdk/sync.d.ts')\"` './dist/nango-sync.d.ts'",
```

Comment:
moving the copying of `sync.d.ts` to postbuild so it happens every time the CLI is built and not just before it is published
Also using `require.resolve` so we don't have to mess with filepath but rely on the dependency location